lib/slax_web/websocket_listener.ex

```diff
@@ -26,9 +26,18 @@ defmodule SlaxWeb.WebsocketListener do
 
     {:ok, pid} =
       :gun.open(:binary.bin_to_list("wss-primary.slack.com"), 443, %{
+        connect_timeout: 60000,
+        retry: 10,
+        retry_timeout: 300,
         transport: :tls,
         protocols: [:http],
-        tls_opts: [verify: :verify_none]
+        http_opts: %{version: :"HTTP/1.1"},
+        tls_opts: [
+          verify: :verify_none,
```

Comment:
I'd be surprised if `:verify_none` gets rid of the error... isn't the lack of verification what the error is complaining about?

Reply:
Yeah I was following the option for connecting with TLS/SSL but without caring to verify the host, at least at first so like that we don't have to setup a cacerts file